package.json

```diff
@@ -2,16 +2,21 @@
     "name": "greenworks",
```

Comment:
Can you please also use `@greenworksjs/greenworks` as name since it will be published there on npm